react/components.md

```diff
@@ -6,15 +6,17 @@ This is an example of a component:
 // Header.jsx
 import React from 'react'
 
-export default React.createClass({
+class Header extends React.Component {
   render() {
     return (
     <div>
       <h1>Header</h1>
     </div>
     )
   }
-})
+}
+
+export default Header
```

Comment:
I think I like `export default class Header extends React.Component {` over declaring and exporting the class separately (especially when we're exporting `default`). Thoughts?